girder/models/user.py

```diff
@@ -46,6 +47,11 @@ def initialize(self):
         self.exposeFields(level=AccessType.ADMIN, fields=(
             'size', 'email', 'groups', 'groupInvites'))
 
+        events.bind('model.user.save.created', 'grantSelfAccess',
+                    self._grantSelfAccess)
+        events.bind('model.user.save.created', 'addDefaultFolders',
+                    self._addDefaultFolders)
```

Comment:
Our convention for event binding names has always been to make them prefixed with the name of the plugin. Now that we have them in core, should we change these names to e.g. `core.grantSelfAccess`?